### ISSUE TITLE
Deadlinks in MonomialAlgebras package

### ISSUE THREAD

**Issue by mahrud** [Macaulay2/M2#1355]
A numbe of links on the MonomialAlgebras package are broken. Here is the link: http://www2.macaulay2.com/Macaulay2/doc/Macaulay2-1.15/share/doc/Macaulay2/MonomialAlgebras/html/

![image](https://user-images.githubusercontent.com/3147718/86103654-a8479600-ba82-11ea-94ed-290ada311243.png)

The links here seem to be working: https://www.mathematik.uni-kl.de/~boehm/Macaulay2/MonomialAlgebras/html/index.html

**Comment by DanGrayson** [comment]
I've just fixed the ones to the journal: 9d90a02102a304c8fe05ecdfe9c09e552445591e

**Comment by mahrud** [comment]
@DanGrayson there are 17 other packages with incorrect journal links:
```
[mahrud@noether M2]$ grep -R "j-sag.org/Volume"  Macaulay2/packages/ | grep "volume URI"
Macaulay2/packages/VersalDeformations.m2:	 "volume URI" => "http://j-sag.org/Volume4/"
Macaulay2/packages/EdgeIdeals.m2:		"volume URI" => "http://j-sag.org/Volume1/"
Macaulay2/packages/PieriMaps.m2:	       "volume URI" => "http://j-sag.org/Volume1/"
Macaulay2/packages/HodgeIntegrals.m2:	"volume URI" => "http://j-sag.org/Volume2/"
Macaulay2/packages/Nauty.m2:	 "volume URI" => "http://j-sag.org/Volume3/"
Macaulay2/packages/ToricVectorBundles.m2:	 "volume URI" => "http://j-sag.org/Volume2/"
Macaulay2/packages/ModuleDeformations.m2:	  "volume URI" => "http://j-sag.org/Volume2/"
Macaulay2/packages/KustinMiller.m2:	     "volume URI" => "http://j-sag.org/Volume4/"
Macaulay2/packages/InvariantRing.m2:	     "volume URI" => "http://j-sag.org/Volume5/"
Macaulay2/packages/PHCpack.m2:	"volume URI" => "http://j-sag.org/Volume5/"
Macaulay2/packages/GraphicalModels.m2:	  "volume URI" => "http://j-sag.org/Volume5/"
Macaulay2/packages/NumericalAlgebraicGeometry.m2:	  "volume URI" => "http://j-sag.org/Volume3/"
Macaulay2/packages/Polyhedra.m2:	 "volume URI" => "http://j-sag.org/Volume1/"
Macaulay2/packages/OldPolyhedra.m2:	 "volume URI" => "http://j-sag.org/Volume1/"
Macaulay2/packages/Binomials.m2:	     "volume URI" => "http://j-sag.org/Volume4/"
Macaulay2/packages/OldToricVectorBundles.m2:	 "volume URI" => "http://j-sag.org/Volume2/"
Macaulay2/packages/QuillenSuslin.m2:	     "volume URI" => "http://j-sag.org/Volume5/"
Macaulay2/packages/MonomialAlgebras.m2:	     "volume URI" => "http://j-sag.org/Volume5/"
```

**Comment by DanGrayson** [comment]
Maybe I should just ask the journal to stop breaking URLs.

**Comment by mahrud** [comment]
How come neither of the two html validators in the repository detect when this happened?

**Comment by DanGrayson** [comment]
They don't check external links.

**Comment by d-torrance** [comment]
The JSAG links were fixed in #3201.

**Comment by mahrud** [comment]
It would be good to occasionally run the html validator on external links, perhaps replacing them with web.archive.org links like wikipedia.
